project-metrics/metrics_service/server.py

```diff
@@ -28,5 +28,16 @@ def recompute(metric_cls_name):
   return 'Successfully recomputed %s.' % metric_cls_name, status.HTTP_200_OK
 
 
+@app.route('/api/metrics')
```

Comment:
super-nit (no reason to create a new PR for this) - prefixing your API endpoints with a version number (`/api/v0/metrics`) is a good habit for the future that can save you a lot of headaches. For this self-contained project, you'll probably never get blocked by this. Just a good habit to form for when you work on larger projects

Reply:
Noted, I noticed this in the release dashboard API. Will keep this in mind going forward.